### ISSUE TITLE
Challenge 20 - Correcting biases in wildfire satellite observations with machine learning

### ISSUE THREAD

**Issue by EsperanzaCuartero** [ECMWFCode4Earth/challenges_2023#4]
# Challenge 20 - Correcting biases in wildfire satellite observations with machine learning

> **Stream 2** - Machine Learning for Earth Science 

### Goal
Develop an application using ML for evaluating and correcting biases between active fire observations from different satellite sensors.

### Mentors and skills
* **Mentors:** Mark Parrington, Miha Razinger, Johannes Kaiser, Mihai Alexe, Siham El Garroussi 
* **Skills required:**
  * Python programming (numpy, pandas, xarray, matplotlib, cartopy …)
  * Machine learning model development and validation (scikit-learn, Pytorch, Tensorflow and/or Jax)
  * Processing and visualization of scientific data (netCDF, HDF, GRIB, …)

<br>

> <b> *Note: Only nationals from European Union (EU) Member States and countries associated with EU’s Space Programme (currently Iceland and Norway) are eligible to participate (see [Terms and Conditions](https://codeforearth.ecmwf.int/terms-and-conditions)).* </b>

<hr>

### Challenge description
The CAMS Global Fire Assimilation System (GFAS) assimilates satellite-based observations of the infrared radiation emitted by large vegetation fires to produce daily and hourly estimates of biomass burning emissions. The observations are expressed as radiative heat release rate, i.e. fire radiative power (FRP), which is linked to the combustion rate.

Global satellite observations of fire radiative power (FRP), provided on a regular 0.1 degree x 0.1 degree grid by GFAS will be available for the project.

Fire emission estimation in GFAS has historically been based on observations from the two MODerate resolution Imaging Spectroradiometer (MODIS) instruments on the NASA Terra and Aqua satellites. In recent years GFAS has been developed to utilise FRP observations from other satellite sensors with different characteristics. Quantifying the biases between different observations is essential to the combined use of the different sensors for estimating fire emissions, taking into account spatial and temporal differences.

A crude bias correction has been developed based on scaling factors which allow the annual emission budget to be maintained irrespective of the sensor(s) that are used in the estimation. The limitations of this approach is that it doesn’t account for the dependence of the biases on region, season, land-cover, fuel condition, observation geometry, FRP magnitude and meteorological conditions. Furthermore, long-term evaluation is required for each new sensor to estimate annual emissions and scaling factors. 

The aim of this project is to develop a technique based on machine learning to evaluate the consistency of FRP data from different sensors over time, taking into account spatial coverage, in order to identify biases and provide a correction that could be used operationally to provide self consistent fire emissions. Figure 1 shows an example of potential differences between MODIS and VIIRS FRP observations for different land cover types.

The project will be divided into the following stages:

1) Data sampling, and bias characterisation: Characterise geographical and temporal distribution of satellite FRP observations (provided on regular 10km x 10km grid) from different sensors and identify biases at different scale. 
2) Bias correction: Characterise correction factors for identified biases in the satellite observations using advanced machine learning techniques such as deep learning and generative adversarial networks.
3) Validation: Validation of the corrected observations to determine the accuracy and effectiveness of the correction.

A more sophisticated approach has been developed that clearly demonstrates, characterises and corrects bias dependence on observation geometry and FRP magnitude (Kaur et al. Remote Sensing 2019). It uses a percentile matching approach, and this project can be understood as an extension of the approach for further dependencies using the power of machine learning.

![pic_challenge4](https://user-images.githubusercontent.com/46716581/221345339-3692bdb4-6a0f-48d7-800b-d7178e37d613.png)

Figure 1: Average gridded FRP between co-located MODIS/Aqua and VIIRS/Suomi-NPP gridded-FRP observations with and without correction classified according to the land cover type (from Figure 5(a) of Kaur et al., Remote Sensing, 2019).
